src/screens/ChatScreen/components/ChatMessageDate.js

```diff
@@ -14,6 +14,8 @@ const styles = theme => ({
   },
   dateView: {
     padding: 8,
+    marginTop: 2,
```

Comment:
By this adds spacing 

**Before**

<img width="227" alt="Screenshot 2021-11-10 at 8 46 55 AM" src="https://user-images.githubusercontent.com/64252451/141043501-3cadb3b9-9645-4d3b-a952-438d5e39cb22.png">


**After**

<img width="227" alt="Screenshot 2021-11-10 at 8 46 46 AM" src="https://user-images.githubusercontent.com/64252451/141043494-cb5f3ea5-3239-4841-ae86-250da6120406.png">